app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt

```diff
@@ -73,7 +73,7 @@ import javax.inject.Singleton
 
 /** Tests for [StoryActivity]. */
 @RunWith(AndroidJUnit4::class)
-@LooperMode(LooperMode.Mode.PAUSED)
+@LooperMode(value = LooperMode.Mode.PAUSED)
```

Comment:
We dont need named argument for this line.

Reply:
@jkugsiya Please reply to this comment.

Reply:
https://github.com/oppia/oppia-android/pull/2903/commits/8ba9f9b8e206dc434c26d0e03a846bc8081e1a09
This requested change is satisfied in the above linked commit.